Tasks/CacheV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,12 @@
+{
+  "loc.friendlyName": "Cache - Invite Only",
+  "loc.helpMarkDown": "Improve build performance by caching files, like dependencies, between pipeline runs.",
+  "loc.description": "Cache files between runs",
+  "loc.instanceNameFormat": "Cache",
```

Comment:
Should we add Invite only in the instance name as well?? @willsmythe  @owenhuynMSFT

Reply:
No need. What we have is fine.